modules/setting/setting.go

```diff
@@ -25,6 +25,9 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	// should be "strk.kbt.io/projects/go/libravatar"
+	"github.com/strk/go-libravatar"
```

Comment:
No comments needed at all and please put after line 23 to group external imports together.